examples/two-tier/main.tf

```
@@ -108,7 +108,7 @@ resource "aws_instance" "web" {
   connection {
     # The default username for our AMI
     user = "ubuntu"
-
+    host = self.public_ip
```

Hi @IPvSean 👋 Thanks for submitting this change. We would like to keep the examples both Terraform 0.11 and Terraform 0.12 compatible where possible for the next few months until the community has much broader adoption of Terraform 0.12. To that end, could you please use the string interpolation syntax here? I verified it works on both Terraform 0.11 and 0.12. Thanks again.

```suggestion
    host = "${self.public_ip}"
```